hack/chglog/CHANGELOG.tpl.md

```diff
@@ -0,0 +1,22 @@
+{{ range .Versions }}
```

Comment:
not sure how to comment on that moved file, but should `CHANGELOG.md` contain info for all the releases now?

Reply:
We can I think for now. But github has a max size it will render for markdown. So one day we will have to come up with a way to fix that if we do all changes.

Reply:
In thinking this through some more if we use one global file, then we still need to keep it in sync between release branches and master. If we have `CHANGELOG.md` only be relevant for the current release we won't have to do that since we will generate the file directly from the release branch.